js/templates/search.es6.js

```diff
@@ -1,14 +1,11 @@
 const bel = require('bel');
 
 module.exports = function () {
-    return bel`<div class="js-search js-menu-section">
-        <form class="js-search-form" name="x">
-          <input type="text" autocomplete="off" placeholder="Search DuckDuckGo" name="q" class="js-search-input" value="${this.model.searchText}" />
```

Comment:
removing `autocomplete="off"` definitely has an effect:

<img width="313" alt="screen shot 2017-05-22 at 4 49 30 pm" src="https://cloud.githubusercontent.com/assets/333669/26327813/bba8f7c2-3f0e-11e7-998e-c4abfc28df1b.png">

Must be

https://stackoverflow.com/questions/15738259/disabling-chrome-autofill

I think we should keep it in there